src/Microsoft.AspNetCore.Razor.Evolution/DefaultDirectiveIRPass.cs

```diff
@@ -2,42 +2,91 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using Microsoft.AspNetCore.Razor.Evolution.Intermediate;
 using Microsoft.AspNetCore.Razor.Evolution.Legacy;
 
 namespace Microsoft.AspNetCore.Razor.Evolution
 {
-    internal class DefaultDirectiveIRPass : RazorIRPassBase
+    internal class DefaultDirectiveIRPass : RazorIRPassBase, IRazorDirectiveClassifierPass
     {
-        public override int Order => RazorIRPass.DefaultDirectiveClassifierOrder;
+        public override int Order => 25;
```

Comment:
> Make this 0. We all agree

We need `RazorDesignTimeIRPass` which in the same phase to run before this pass. So setting this to 25.

@rynowak @NTaylorMullen